corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -398,19 +455,61 @@ def create_report(self):
         report = ReportConfiguration(
             domain=self.domain,
             config_id=data_source_config_id,
-            title=self.cleaned_data['report_name'],
+            title=self.report_name,
             aggregation_columns=self._report_aggregation_cols,
             columns=self._report_columns,
             filters=self._report_filters,
-            configured_charts=self._report_charts
+            configured_charts=self._report_charts,
+            report_meta=ReportMeta(
+                created_by_builder=True,
+                builder_report_type=self.report_type
+            )
         )
         report.validate()
         report.save()
         return report
 
     @property
+    @memoized
+    def initial_filters(self):
+        if self.existing_report:
+            return [self._get_view_model(f) for f in self.existing_report.filters]
+        if self.source_type == 'case':
+            return [
+                {'property': 'closed', 'display_text': 'closed', 'format': 'Choice'},
+                # TODO: Allow users to filter by owner name, not just id.
+                # This will likely require implementing data source filters.
+                {'property': 'owner_id', 'display_text': 'owner id', 'format': 'Choice'},
+            ]
+        else:
+            # self.source_type == 'form'
+            return [
+                {'property': 'timeEnd', 'display_text': 'Form completion time', 'format': 'Date'}
+            ]
+
+    def _get_view_model(self, filter):
+        """
+        Given a ReportFilter, return a dictionary representing the knockout view
+        model representing this filter in the report builder.
+        """
+        filter_type_map = {
+            'dynamic_choice_list': 'Choice',
+            'choice_list': 'Choice',  # This exists to handle the `closed` filter that might exist
+            'date': 'Date',
+            'numeric': 'Numeric'
+        }
+        return {
+            'property': self._get_property_from_column(filter['field']),
+            'display_text': filter['display'],
+            'format': filter_type_map[filter['type']]
+        }
+
+    def _get_property_from_column(self, col):
+        return self._properties_by_column[col]['id']
+
+    @property
     def _report_aggregation_cols(self):
-        return []
+        return ['doc_id']
```

Comment:
Correct. This property is overridden by the subclasses, like here for example: https://github.com/dimagi/commcare-hq/blob/report-builder-2/corehq/apps/userreports/reports/builder/forms.py#L591-L595